connection/connection.go

```diff
@@ -53,170 +53,99 @@ func pipe(c1, c2 PipeConnection) {
 	}
 }
 
-func CreateBasicPlayerConnection(conn Connection, remoteAddr net.Addr) *BasicPlayerConnection {
```

Comment:
I think NewBasicPlayerConn(...) would be less verbose

Reply:
Yea the New is better than the Create, good one